doc/tasks/composer_normalize.md

```diff
@@ -0,0 +1,43 @@
+# Composer Normalize
+
+If you are using `composer`, you have probably modified the file `composer.json` at least once to keep things nice
+and tidy.
+
+This task is a wrapper around a composer plugin for tidying up the file `composer.json`.
+
+The default configuration looks like:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        composer_normalize:
+            indent_size: 4
+            indent_style: space
+            update_lock: true
+            verbose: false
+```
+
+**indent_size**
+
+*Default: 4*
+
+Indent size (an integer greater than 0); should be used with the `indent_style` option
+
+**indent_style**
+
+*Default: space*
+
+Indent style (one of "space", "tab"); should be used with the `indent_size` option
+
+**update_lock**
+
+*Default: false*
```

Comment:
Should this be `true`?

See `GrumPHP\Task\ComposerNormalize::getConfigurableOptions()`.

Reply:
Fixed.